src/PubSub/PubSubClient.php

```diff
@@ -357,6 +357,8 @@ function (array $subscription) {
      * @param Subscription $subscription The subscription to take a snapshot of.
      * @param array $options [optional] Configuration options.
      * @return Snapshot
+     * @experimental
+     * @whitelisted
```

Comment:
I have added annotations here, but was not sure about the `snapshot` method at https://github.com/GoogleCloudPlatform/google-cloud-php/pull/539/files#diff-6e95b2a481762bdbf45a07703f71ca5cR386, which doesn't have the whitelisted description.